docs/notebooks/Berger_POPC_25ns_buildH_calcOP.ipynb

```diff
@@ -0,0 +1,1394 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>76</th>\n",
+       "      <td>oleoyl_C16b</td>\n",
+       "      <td>-0.07041</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>77</th>\n",
+       "      <td>oleoyl_C17a</td>\n",
+       "      <td>-0.05500</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>78</th>\n",
+       "      <td>oleoyl_C17b</td>\n",
+       "      <td>-0.05225</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>79</th>\n",
+       "      <td>oleoyl_C18a</td>\n",
+       "      <td>0.03903</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>80</th>\n",
+       "      <td>oleoyl_C18b</td>\n",
+       "      <td>-0.05292</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>81</th>\n",
+       "      <td>oleoyl_C18c</td>\n",
+       "      <td>-0.05025</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "        CH_name       OP\n",
+       "49   oleoyl_C2a -0.15474\n",
+       "50   oleoyl_C2b -0.17336\n",
+       "51   oleoyl_C3a -0.16911\n",
+       "52   oleoyl_C3b -0.17847\n",
+       "53   oleoyl_C4a -0.17385\n",
+       "54   oleoyl_C4b -0.18235\n",
+       "55   oleoyl_C5a -0.17504\n",
+       "56   oleoyl_C5b -0.17690\n",
+       "57   oleoyl_C6a -0.16138\n",
+       "58   oleoyl_C6b -0.16692\n",
+       "59   oleoyl_C7a -0.15512\n",
+       "60   oleoyl_C7b -0.15994\n",
+       "61   oleoyl_C8a -0.10190\n",
+       "62   oleoyl_C8b -0.10603\n",
+       "63   oleoyl_C9a -0.08281\n",
+       "64  oleoyl_C10a -0.01791\n",
+       "65  oleoyl_C11a -0.05119\n",
+       "66  oleoyl_C11b -0.04599\n",
+       "67  oleoyl_C12a -0.09275\n",
+       "68  oleoyl_C12b -0.08879\n",
+       "69  oleoyl_C13a -0.08643\n",
+       "70  oleoyl_C13b -0.08259\n",
+       "71  oleoyl_C14a -0.09133\n",
+       "72  oleoyl_C14b -0.08893\n",
+       "73  oleoyl_C15a -0.07729\n",
+       "74  oleoyl_C15b -0.07524\n",
+       "75  oleoyl_C16a -0.07256\n",
+       "76  oleoyl_C16b -0.07041\n",
+       "77  oleoyl_C17a -0.05500\n",
+       "78  oleoyl_C17b -0.05225\n",
+       "79  oleoyl_C18a  0.03903\n",
+       "80  oleoyl_C18b -0.05292\n",
+       "81  oleoyl_C18c -0.05025"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df[[\"CH_name\", \"OP\"]].iloc[49:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot of the order parameter\n",
+    "\n",
+    "Now that we have loaded the data with pandas, and that we can select any part of the lipid, we can use the matplotlib module to make a nice plot. For example, we can plot the order parameter of the palmitoyl chain. Note that we select rows 18 to 45 to show only the first 15 carbons, since we do not want the final CH3 of carbon 16."
```

Comment:
'rows 18 to 45 to show only the first 15 carbons' problème de mise à jour des valeurs ?

Reply:
What I meant we the first 15 carbons of the palmitoyl chain (which has 16 carbons), thus we ignore the final CH3.